### PR TITLE
fix issue #38: show info instead error when there is no tomcat server

### PR DESCRIPTION
**Pull request by SummerSun**

### CODE REVIEW

**Comment by SummerSun** [comment]
@adashen 
Reference the behaviors of VScode itself, Github extension and Docker extension, all shows nothing or just pop up an info bar when trying to removing non exist items.

File: src/extension.ts [hunk 1]
```diff
@@ -79,10 +79,11 @@ async function getTargetServer(tomcat: TomcatController, tomcatItem ?: TomcatSer
     }
 
     if (!server) {
-        return Promise.reject(new Error(Utility.localize('tomcatExt.noserver', 'Tomcat server is undefined')));
-    } else {
-        return server;
+        vscode.window.showInformationMessage(Utility.localize('tomcatExt.noservertodelete', 'No tomcat server.'));
+        return null;
```
Comment (andxu) [inline src/extension.ts]: return null will make the command fail silently.

**Comment by adashen** [comment]
@SummerSun , if user want to run tomcat on some already deleted tomcat server. I think this should be treated as error.

**Comment by SummerSun** [comment]
@adashen check one more time that run on tomcat or debug on tomcat have no logic interaction with getTargetServer. They are different logic blocks.

File: src/extension.ts [hunk 2]
```diff
@@ -77,46 +77,51 @@ async function getTargetServer(tomcat: TomcatController, tomcatItem ?: TomcatSer
...
-    }
+    return server;
 }
 
 async function serverStart(tomcat: TomcatController, tomcatItem ?: TomcatServer): Promise<void> {
     const server: TomcatServer = await getTargetServer(tomcat, tomcatItem);
     if (server) {
         await tomcat.startServer(server);
+    } else {
+        vscode.window.showInformationMessage(Utility.localize('tomcatExt.noserver', 'No tomcat server.'));
```
Comment (andxu) [inline src/extension.ts]: add await

**Pull request merged**